src/tab.group.ts

```diff
@@ -327,12 +329,26 @@ export default class TabGroup extends LitElement {
 
   #onTabSelected(event: Event) {
     if (event.target instanceof Tab && event.target.selected) {
-      this.#selectedTab = event.target;
-      this.#updateSelectedTabAndPanel();
```

Comment:
Now that `#onTabSelected()` is the only place this method is called, there's no need for it as a method. So I moved its logic to this method.